lessons/PlottinginPythonwithMatplotlib/hwk3.2.py

```diff
@@ -0,0 +1,46 @@
+#Import necessary modules for movie animation
+from scipy.io.netcdf import *
+import numpy as np
+import pylab as p
```

Comment:
It would be if we had extra time.  The animation stuff is for more advanced
users.

On Wed, Aug 5, 2015 at 3:04 PM, Elliott Sales de Andrade <
notifications@github.com> wrote:

> In lessons/PlottinginPythonwithMatplotlib/hwk3.2.py
> <https://github.com/UofTCoders/studyGroup/pull/33#discussion_r36341017>:
>
> > @@ -0,0 +1,46 @@
> > +#Import necessary modules for movie animation
> > +from scipy.io.netcdf import *
> > +import numpy as np
> > +import pylab as p
>
> Please avoid pylab.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UofTCoders/studyGroup/pull/33/files#r36341017>.
>